### DISCUSSION TITLE
Example(s) for RAVEN Bayesian Optimization

### DISCUSSION THREAD

**Discussion by JohninSJ** [idaholab/raven#2316] [Q&A]
I have some test data and a heat transfer equation with pre-defined format, for instance Nu = f(C1, C2, C3, Re, Pr), and would like to refine/tune the equation coefficients C1, C2 and C3 by using Bayesian Inference. Is RAVEN Bayesian Optimization applicable for this purpose? If yes, could you please advise where I can find some examples for references.  Much appreciated.

**Comment by wangcj05** [comment]
It seems you are looking for Bayesian model calibration. There is a raven plugin for Bayesian calibration. Please check https://github.com/idaholab/BayCal.
